fulp_modules/lisa/velvet_fu.dm

```diff
@@ -0,0 +1,292 @@
+	A.do_attack_animation(D)
+	var/picked_hit_type = "side kick"
+	var/bonus_damage = 10 // Kicking deals more damage
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 8
+		picked_hit_type = "iron hooved"
+	D.apply_damage(bonus_damage, A.get_attack_type())
+	D.visible_message("<span class='danger'>[A] [picked_hit_type]ed [D]!</span>", \
+					"<span class='userdanger'>You're [picked_hit_type]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [picked_hit_type] [D]!</span>")
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+
+/// Help, Granters & Uplink Item
+/mob/living/proc/velvetfu_help()
+	set name = "Recall Teachings"
+	set desc = "Remember the martial techniques of Velvet-Fu."
+	set category = "VelvetFu"
+
+	to_chat(usr, "<b><i>You try to remember the VHS tapes of Velvet-Fu...</i></b>\n\
+	<span class='notice'>Iron Hoof</span>: Disarm/Grab/Harm while opponent is down, though Disarm works best.\n\
+	<span class='notice'>Flying Axe Kick</span>: Harm Disarm. Deals damage and causes bleeding. Costs 50 Stamina.\n\
+	<span class='notice'>Goat Headbutt</span>: Disarm Grab. Deals brute while stunning your opponent, but hurts you. Costs 60 Stamina.\n\
+	<span class='notice'>Full Thrust</span>: Grab Harm. Deals brute and has a chance to knock your opponent down. Costs 70 Stamina.\n\
+	<span class='notice'>Minor Iris</span>: Harm Harm Harm. Devastatingly slashes your opponent. Costs 80 Stamina.\n\
+	<span class='notice'>Receding Stance</span>: Regenerates 40 Stamina. Requires standing still.\n\
+	<span class='notice'>Twisted Stance</span>: Regenerates 40 Stamina. Deals brute damage.</span>")
+
+/obj/item/book/granter/martial/velvetfu
+	martial = /datum/martial_art/velvetfu
+	name = "Hollywood VHS tape"
+	martialname = "velvet-fu"
+	desc = "A VHS tape labelled 'Grand-Master's Course'. This seems modified, causing it to beam the content straight into your eyes."
+	icon = 'fulp_modules/lisa/sprites/casette.dmi'
+	icon_state = "velvet"
+	greet = "You've finished watching the Velvet-Fu VHS tape."
+	remarks = list("Smooth as Velvet...", "Show me your stance!", "Left Jab!", "Right Jab!", "Kick, kick!", "Ah, so fast...", "Now forget the basics!", "...But remember the style!")
+
+/obj/item/book/granter/martial/velvetfu/onlearned(mob/living/carbon/user)
+	..()
+	if(oneuse == TRUE)
+		desc = "A VHS tape labelled 'Grand-Master's Course'. The film seems ripped out, and can't be put back in."
+		name = "Broken VHS tape"
+		icon_state = "velvet_used"
+
+/datum/uplink_item/stealthy_weapons/velvetfu
+	name = "Velvet-Fu VHS tape"
+	desc = "Velvet-Fu is a knock-off Martial Art straight from Hollywood.\
+			'A VHS tape that teaches YOU, the secrets of Velvet-Fu!' \
+			Now specially modified to beam its knowledge directly into your eyes, removing the need for a TV."
+	item = /obj/item/book/granter/martial/velvetfu
+	cost = 8
```

Comment:
me when

when gloves of the north star

Reply:
uh oh

instant stamcrit

Reply:
> uh oh
> 
> instant stamcrit

not if you only spam disarm 😎